prow/config/jobs/envoy-1.9.yaml

```diff
@@ -0,0 +1,66 @@
+org: istio
+repo: envoy
+branches: [release-1.9]
+image: envoyproxy/envoy-build-ubuntu:f21773ab398a879f976936f72c78c9dd3718ca1e
```

Comment:
This should be `11efa5680d987fff33fde4af3cc5ece105015d04` (matching version from `.bazelrc`).